picklefield/fields.py

```diff
@@ -170,12 +169,8 @@ def pre_save(self, model_instance, add):
         value = super(PickledObjectField, self).pre_save(model_instance, add)
         return wrap_conflictual_object(value)
 
-    if django.VERSION < (2, 0):
-        def from_db_value(self, value, expression, connection, context):
-            return self.to_python(value)
-    else:
-        def from_db_value(self, value, expression, connection):
-            return self.to_python(value)
+    def from_db_value(self, value, expression, connection, context=None):
+        return self.to_python(value)
```

Comment:
This is still necessary unless you drop support for Django 1.11.

Reply:
It works in all versions with full coverage, but the django 3.0 warning is back. Pretty strange because master (3.1) is ok.